roles/autossl/defaults/main.yml

```diff
@@ -1,2 +1,50 @@
 ---
-# defaults file for autossl
+
+## Deployment
+replicas: 1
+image_name: "quay.io/3scale/autossl"
+image_version: "v1.0.0"
+image_pull_secret_name: "quay-pull-secret"
+acme_staging: ""
+contact_email: "3scale-operations@redhat.com"
+proxy_endpoint: "https://multitenant-admin.dev.3sca.net"
+storage_adapter: "redis"
+redis_host: "dev-saas-autossl-ec-redis.ng.0001.use1.cache.amazonaws.com"
+redis_port: "6379"
+verification_endpoint: "https://multitenant-admin.dev.3sca.net/swagger/spec.json"
+log_level: "warn"
+domain_whitelist: "autossl.dev.3sca.net"
+domain_blacklist: ""
+liveness_probe_initial_delay_seconds: 25
+liveness_probe_timeout_seconds: 1
+liveness_probe_period_seconds: 10
+liveness_probe_success_threshold: 1
+liveness_probe_failure_threshold: 3
+readiness_probe_initial_delay_seconds: 25
+readiness_probe_timeout_seconds: 1
+readiness_probe_period_seconds: 10
+readiness_probe_success_threshold: 1
+readiness_probe_failure_threshold: 3
+resources_requests_cpu: "50m"
+resources_requests_memory: "50Mi"
+resources_limits_cpu: "100m"
+resources_limits_memory: "100Mi"
+
+## Service
+external_dns_hostname: "mtssl-edge-a.dev.3sca.net,mtssl-edge-b.dev.3sca.net,autossl.dev.3sca.net"
+load_balancer_proxy_protocol: "*"
+load_balancer_cross_zone_load_balancing_enabled: true
+load_balancer_connection_draining_enabled: true
+load_balancer_connection_draining_timeout: 60
+load_balancer_connection_healthcheck_healthy_threshold: 2
+load_balancer_connection_healthcheck_unhealthy_threshold: 2
+load_balancer_connection_healthcheck_interval: 5
+load_balancer_connection_healthcheck_timeout: 3
+load_balancer_access_log_enabled: true
+load_balancer_access_log_emit_interval: 5
+load_balancer_access_log_s3_bucket_name: "dev-s3-logs"
```

Comment:
Same as before, the bucket should be undefined (so load balancer logs are disabled) or something that requires user input.